configs/BUILD

```diff
@@ -0,0 +1,13 @@
+package(default_visibility = ["//visibility:public"])
+
+genrule(
+    name = "example_configs",
+    srcs = glob(["*.json"]) + ["//examples:configs"],
+    outs = ["example_configs.tar"],
+    cmd = "./$(location configgen.sh) $(@D) $(locations //examples:configs)",
```

Comment:
Will do (though I just followed the example in the bazel docs). AFAIK it actually is always relative. It's actually kind of annoying dealing with this.